pkg/kube/secrets/secrets.go

```diff
@@ -0,0 +1,31 @@
+package secrets
+
+import (
+	"github.com/aquasecurity/starboard/pkg/docker"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewImagePullSecret constructs a new image pull Secret with the specified
+// registry server and basic authentication credentials.
+func NewImagePullSecret(meta metav1.ObjectMeta, server, username, password string) (*corev1.Secret, error) {
```

Comment:
Is this the function for testing?

Reply:
For now it's only used in the integration test, but I'll need to create a secret in the same namespace in which we create the scan Job. Currently the username/password are passed as ENVS, which has to be fixed

Reply:
I see.